src/material-experimental/mdc-chips/chip-grid.ts

```diff
@@ -379,6 +379,10 @@ export class MatChipGrid
    */
   setDescribedByIds(ids: string[]) {
     this._ariaDescribedby = ids.join(' ');
+
+    if (this._chipInput) {
+      this._chipInput.setDescribedByIds(ids);
```

Comment:
I don't think we have to deal with that in this particular method -- any time an error spawns this method will be called with all relevant ids and set the complete value of the property bound to the template.

The window for this to happen is very small:  We assert that a chipInput is present during `ngAfterViewInit()`, so the error that we're wanting to propagate would need to occur and be set before that time.  With this in mind we can handle this case by calling `setDescribedByIds` after the assertion.

The downside of doing this is it means we need to keep the value of `this._ariaDescribedby` up to date in this component as well.  Probably a best practice anyway since it's the one implementing `MatFormFieldControl`.